resources/views/admin/chatrooms/components/meetingList.blade.php

```diff
@@ -8,7 +8,7 @@
                         <td style="width: 50px">{{ $loop->index + 1 }}</td>
                         <td class="text-start">{{ $meeting->title }}</td>
                         <td>{{ $meeting->user->user_name }}</td>
-                        <td>{{ $meeting->date }} {{ $meeting->start_at }} : 00~</td>
+                        <td>{{ $meeting->date }} {{ date('G:i', strtotime($meeting->start_at)) }}~</td>
```

Comment:
I prefer to `H` instead of `G` in time formatting because it's easy to understand that it represents the `Hour`

Reply:
I want to display the time like 6:00 without leading "0"